src/Simulation/QSharpFoundation/Bitwise/Bitwise.qs

```diff
@@ -83,7 +83,17 @@ namespace Microsoft.Quantum.Bitwise {
     /// let x = Parity(a); // x : Int = 1.
     /// ```
     function Parity (a : Int) : Int {
-        body intrinsic;
+        mutable tmpParam = a;
+        mutable result = false;
+        while tmpParam != 0 {
+            // Clear (to `0`) the least significant `1` (not to confuse with the least significant bit). E.g.
```

Comment:
```suggestion
            // Clear (set to `0`) the least significant `1` (not to be confused with the least significant bit). E.g.
```

Reply:
I'm not opposing to the expression "set to `0`", just want to make sure that the decision is fully informed.

At my first job I was reading a microcontroller datasheet. In all cases there was written there either "clear (to '0')" or "set (to '1')". Since then it seemed to me very clear when the documentation was written like that. And if in any single case it was written just "clear" or "set", then I still was clearly understanding what they meant.
Later on the other datasheets were mentioning something like "to reset" means "to set to a power-on-reset value" (which for one entities/bits meant '0', for the others meant '1'). And it also seemed to me very clear, even if they were writing just "reset".
The expression "set to `0`" may sound slightly confusing or odd for those who have similar background as mine.

But if you have strong preference regarding that, then there is absolutely no problem for me, I'll make that change.

Reply:
Not applicable any more. Code fragment has been replaced.